lib/vsc/mympirun/rm/sched.py

```diff
@@ -145,29 +93,6 @@ def __init__(self, options=None, **kwargs):
 
         super(Sched, self).__init__(**kwargs)
 
-    # factory methods for Sched. To add a new Sched class just create a new class that extends the cluster class
-    # see http://stackoverflow.com/questions/456672/class-factory-in-python
-    @classmethod
-    def _is_sched_for(cls, name=None):
-        """see if this class can provide support for sched class"""
-        if name is not None:
-            # add class name as default
-            return name in cls._sched_for + [cls.__name__]
-
-        # guess it from environment
-        totest = cls._sched_environ_test
-        if cls.SCHED_ENVIRON_ID is not None:
-            totest.append(cls.SCHED_ENVIRON_ID)
-
-        for envvar in totest:
-            envval = os.environ.get(envvar, None)
-            if not envval:
-                continue
-            else:
-                return True
-
-        return False
-
     # other methods
     def set_sched_id(self):
         """get a unique id for this scheduler"""
```

Comment:
please clarify docstring that this is the *job* ID

Reply:
now i see why i missed it. this is old code. will fix